src/System.Transactions.Local/src/System/Transactions/TransactionsEtwProvider.cs

```diff
@@ -73,7 +73,7 @@ internal enum TraceSourceType
     [EventSource(
         Name = "System.Transactions.TransactionsEventSource",
         Guid = "8ac2d80a-1f1a-431b-ace4-bff8824aef0b",
-        LocalizationResources = "FxResources.System.Transactions.SR")]
+        LocalizationResources = "FxResources.System.Transactions.Local.SR")]
```

Comment:
As an aside and for a separate PR, do we have any tests that exercise the EventSource, make sure it's generating valid events, make sure it's not generating error events, etc.?  That'd be good to have.  We do this for other EventSources in corefx, if you need examples.